rinja_derive/src/generator.rs

```diff
@@ -25,9 +25,12 @@ pub(crate) fn template_to_string(
     input: &TemplateInput<'_>,
     contexts: &HashMap<&Arc<Path>, Context<'_>, FxBuildHasher>,
     heritage: Option<&Heritage<'_, '_>>,
-    target: Option<&str>,
+    tmpl_kind: TmplKind,
 ) -> Result<usize, CompileError> {
-    let ctx = &contexts[&input.path];
+    if tmpl_kind == TmplKind::Struct {
+        buf.write("const _: () = { extern crate rinja as rinja;");
```

Comment:
Is the `as rinja` part useful?

Reply:
The same line `extern crate rinja as rinja;` already exists at other places outside of this PR's changes.

Right now it's a placeholder. I plan to add a `crate_` argument to `#[template]`. That would make it easier to derive Templates in a macro.